capsules/src/virtual_uart.rs

```diff
@@ -52,9 +52,6 @@ use kernel::hil::uart;
 use kernel::utilities::cells::{OptionalCell, TakeCell};
 use kernel::ErrorCode;
 
-const RX_BUF_LEN: usize = 64;
```

Comment:
This should be left here, as the recommended rx buf len from the author of this capsule.

Reply:
Yes, that makes sense.